tests/actors/test_power_meter.py

```diff
@@ -146,82 +149,58 @@ async def test_power_meter_periodic_update(request: pytest.FixtureRequest) -> No
                 f"wait for PowerMeter periodic update [{tt.Name}]"
             )
 
-
-# @pytest.mark.asyncio
-# async def test_power_meter_aggregate_power_forward(tmp_path, monkeypatch, request):
+@pytest.mark.asyncio
+async def test_async_power_update(request: pytest.FixtureRequest):
 #     """Verify that when a simulated change in power is generated, Scadd and Atn both get a PowerWatts message"""
+    async with ScadaLiveTest(
+        request=request,
+    ) as h:
+        h.start_child1() # start primary scada
+        h.start_parent() # start atn
+        scada = h.child1_app.scada
+        
+
+        data = typing.cast(ScadaData, scada._data)
+        print(f"type of h.child1_app.scada is {type(scada)}")
+        atn_received_counts = h.parent_to_child_stats.num_received_by_type
+        initial = atn_received_counts['power.watts']
+        print(f"atn has received {initial} power.watts messages")
+        await h.await_for(
+                lambda: scada.data.latest_power_w is not None,
+                "Scada wait for initial PowerWatts"
+            )
+        print(f"scada.data.latest_power_w is {scada.data.latest_power_w}")
+
+        p = typing.cast(PowerMeterDriverThread, h.child1_app.proactor._communicators[H0N.primary_power_meter]._sync_thread)
```

Comment:
There's a get_communicators() accessor; there might also be an accessor which takes a type and raises an exception if the requested object isn't that type. If there's not perhaps there should be. My preferences is to use public functions where possible because it cuts down on warning noise.